cmd/launcher/options.go

```diff
@@ -189,6 +189,7 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		EnrollSecretPath:                   *flEnrollSecretPath,
 		InsecureTLS:                        *flInsecureTLS,
 		InsecureTransport:                  *flInsecureTransport,
+		KolideHosted:                       *flKolideHosted,
```

Comment:
@directionless , do we need to update the installer to make sure this flag is set to true for Kolide customers?

Reply:
Not Yet :)

My plan is that once some of the stuff goes out of beta, match against the server URL and auto set it